conf/defaults.yaml

```diff
@@ -281,3 +281,16 @@ pacemaker.thread.timeout: 10
 pacemaker.childopts: "-Xmx1024m"
 pacemaker.auth.method: "NONE"
 pacemaker.kerberos.users: []
+
+storm.resource.isolation.plugin: "org.apache.storm.container.cgroup.CgroupManager"
+
+# Configs for CGroup support
+cgroup.storm.hierarchy.dir: "/cgroup/storm_resources"
+cgroup.storm.resources:
+    - cpu
+    - memory
+cgroup.storm.hierarchy.name: "storm"
+# Also determines whether the unit tests for cgroup runs.  If cgroup.enable is set to false the unit tests for cgroups will not run
+cgroup.enable: false
+cgroup.supervisor.rootdir: "storm"
+cgroup.cgexec.cmd: "/bin/cgexec"
```

Comment:
Can we please prefix these configs with "storm."  and probably also "storm.supervisor."